packages/visual-dev/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styled from 'styled-components'
+import { Icon } from "../Icons";
+
+import * as Styles from './Styles'
+
+interface BadgeProps {
+    status: "info" | "success" | "active" | "critical" | "warning";
+    children?: any[] | string;
+    icon?: string;
```

Comment:
It could be nice to have `icon` be a boolean as well. If `true` display some default icon, if `false`/empty string/undefined display no icon, if a string display a custom icon.

Reply:
This update would be amazing 🙏🏻. Switch statement should be enough for setting default icons.